pol-core/pol/target.cpp

```diff
@@ -157,6 +138,13 @@ void TargetCursor::cancel( Mobile::Character* chr )
 
 void TargetCursor::handle_target_cursor( Mobile::Character* chr, PKTBI_6C* msg )
 {
+  u32 target_cursor_serial = cfBEu32( msg->target_cursor_serial );
+  if ( target_cursor_serial != cursorid_ )
+  {
+    SuspiciousActs::OutOfSequenceCursor( chr->client );
```

Comment:
I think CUO would then also have the suspicicousact when by script CancelTarget is called. Since it sends as cursor_serial in this case a 0.